README.md

```diff
@@ -2,73 +2,76 @@
 
 
 ## Overview
-
 OpenFin FDC3 API uses the message Channels framework to expose its API to consuming applications.  You can see the documentation for these APIs here:  https://cdn.openfin.co/docs/javascript/stable/InterApplicationBus.Channel.html.
 
-This project consist of 2 parts:
-1. The FDC3 Provider, taking care of intents, context and resolving them (UI)
+This project consist of 3 parts:
+1. The FDC3 Service, taking care of intents, context and resolving them (UI)
 2. The FDC3 Client, exposing API's for applications to handle/raise intents with contexts
 
 ### Dependencies
-- OpenFin version >= 8.56.30.42 
-- RVM >= 4.2.0.33 
+- OpenFin version for applications using FDC3 = 9.61.38.41
+- OpenFin version used in the FDC3 Service = 10.66.39.43
+- RVM >= 4.7
 
 ### Features
 * Raise an FDC3 Intent
 * Resolve an FDC3 Intent
 * Open an application with an intent/context
 * Attach listeners for Intents and Contexts
-
+* Context channel support, to filter context broadcasts
 
 ### API Documentation
-
-[official API spec](https://github.com/FDC3/API)
-
-
-## Roadmap
-This is a WIP living implementation of the FDC3 API.
-
-### Usage
-
-An in-depth usage guide and additional documentation will be published in due course.
-
-## Run Locally
-
-To preview the functionality of the service without integrating it into an existing application - or to start contributing to the service - the service can be ran locally. By checking out this repo and then running the project.
+[Official API spec](hhttps://fdc3.finos.org/)
 
 ## Getting Started
 
-Using the FDC3 service is done in two steps, add the service to application manifest and import the API:
+Integrating the FDC3 service is done in two steps, add the service to application manifest and import the API:
```

Comment:
"Integrating the FDC3 service is done in two steps - add the service to application manifest, and import the API:"

Reply:
Aligned punctuation with layouts